/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.cmucs426.model.test;

/**
 *
 * @author minh
 */
import com.mycompany.user.User;
import junit.framework.TestCase;

public class UserTest extends TestCase {
    public void testUserConstructor() {
        User user = new User("user1", "user1@cmucs246.com");
        // Kiểm tra xem email có đúng là "user1@cmucs246.com" không
        assertEquals("user1@cmucs246.com", user.getEmail());
    }

    public void testSetEmail() {
        User user = new User("user1", "user1@cmucs246.com");
        user.setEmail("newemail@cmucs246.com");
        assertEquals("newemail@cmucs246.com", user.getEmail());
    }

    public void testUpdateEmail() {
        User user = new User("user1", "user1@cmucs246.com");
        user.updateEmail("updatedemail@cmucs246.com");
        assertEquals("updatedemail@cmucs246.com", user.getEmail());
    }
}
